src/MusicalScore/Graphical/DrawingParameters.ts

```diff
@@ -9,6 +18,45 @@ export class DrawingParameters {
     public drawComments: boolean;
     public drawMarkedAreas: boolean;
 
+    public static DrawingParametersStringToEnum(stringParameter: string): DrawingParametersEnum {
```

Comment:
Don't know if we need this. Maybe it is possible to use string based enums here? Those can be used from JS as well. Introduced in 2.4: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-4.html

Reply:
We can use string based enums, but not if we port it back to C#.
I like enums as numbers (ideally ints), strings are messy.

Reply:
https://goo.gl/HUdLTh

What about this? It doesn't seem too messy to me, and you can use it directly from JS with the string value.